custom/icds_reports/migrations/sql_templates/database_views/bihar_vaccine.sql

```diff
@@ -11,6 +11,8 @@ CREATE VIEW bihar_vaccine_view AS
         "bihar_demographics"."mother_name" AS "mother_name",
         "bihar_demographics"."father_id" AS "father_id",
         "child_health"."mother_case_id" AS "mother_id",
+        "child_health"."delivery_nature" AS "delivery_nature",
+        COALESCE("child_health"."term_days" > 0, NULL) AS "term_days",
```

Comment:
hey, term days has to be a numeric no? the difference. THis looks like it will be either True/False/Null.

Reply:
thought we need to return true or false..corrected